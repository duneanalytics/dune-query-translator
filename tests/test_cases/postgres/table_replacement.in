SELECT * FROM erc20."ERC20_evt_Transfer", erc721.ERC721_evt_Transfer as x, aave."AaveCollateralVaultProxy_evt_DeployVault"
