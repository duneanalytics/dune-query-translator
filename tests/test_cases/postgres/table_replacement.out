SELECT
  *
FROM erc20_optimism.evt_Transfer, erc721_ethereum.evt_Transfer AS x, aave_optimism.AaveCollateralVaultProxy_evt_DeployVault